python/ray/tune/schedulers/pbt.py

```diff
@@ -231,8 +231,14 @@ def on_trial_add(self, trial_runner, trial):
         self._trial_state[trial] = PBTTrialState(trial)
 
     def on_trial_result(self, trial_runner, trial, result):
-        if self._time_attr not in result or self._metric not in result:
-            return TrialScheduler.CONTINUE
+        if self._time_attr not in result:
+            raise TuneError("Cannot find time_attr {} in trial results. "
+                            "Make sure that this attribute is returned "
+                            "in the results of your Trainable.")
```

Comment:
Can we also list the current attributes the user is returning?

Reply:
fixed in latest commit

Reply:
Hm did you push? I don't see it